File: 240116/좀 더 어려운 수학 점수/math-scores-are-more-difficult.py
A = input().split()
B = input().split()

aMath = int(A[0])
aEng = int(A[-1])

bMath = int(B[0])
bEng = int(B[-1])


if aMath > bMath:
    print("A")
elif aMath == bMath:
    if aEng > bEng:
        print("A")
    else:
        print("B")
else:
    print("B")

File: 240409/좀 더 어려운 수학 점수/math-scores-are-more-difficult.swift
let arrA = readLine()!.split(separator: " ").map {Int($0)!}
let arrB = readLine()!.split(separator: " ").map {Int($0)!}
let (aMath, aEng) = (arrA[0], arrA[1])
let (bMath, bEng) = (arrB[0], arrB[1])

if aMath > bMath {
    print("A")
} else if bMath > aMath {
    print("B")
} else {
    if aEng > bEng {
        print("A")
    } else {
        print("B")
    }
}